README.md

```diff
@@ -32,3 +32,90 @@ $ scala-cli run ./examples/src/main/scala/example.scala --native --native-compil
 Bindings generated using [sn-bindgen](https://sn-bindgen.indoorvivants.com/)
 
 ![GtkBindingsDemo](https://user-images.githubusercontent.com/1052965/217792578-a4faa191-053b-48a7-aa2a-207709690fa1.gif)
+
+## GIO Resource Integration
```

Comment:
Does this resource integration have additional benefits on top of Scala Native's [basic resource embedding](https://scala-native.org/en/stable/lib/javalib.html#embedding-resources)?

I.e. are there GLib/Gnome/etc. methods that take, for example, a resource name, thus providing deeper integration?